### PR TITLE
[16.0][ADD] pos_sale_product_config_no_variant

### PR DESCRIPTION
**Pull request by sandipvyas-osi**

### CODE REVIEW

**Comment by sandipvyas-osi** [comment]
@dreispt Can you please review this PR?

File: pos_sale_product_config_no_variant/models/pos_order.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+class PosOrder(models.Model):
+    _inherit = "pos.order"
+
+    def _get_fields_for_order_line(self):
+        fields = super(PosOrder, self)._get_fields_for_order_line()
+        fields.extend(
+            [
+                "product_no_variant_attribute_value_ids",
+            ]
+        )
```
Comment (dreispt) [inline pos_sale_product_config_no_variant/models/pos_order.py:16]:
```suggestion
        fields.extend(["product_no_variant_attribute_value_ids"])
```

**Comment by sandipvyas-osi** [comment]
@OCA-git-bot 

Can you please review this PR and merge the same?

Thanks

**Review by dreispt — APPROVED** [review]

**Pull request merged**